generated-dockerfiles/alpine/10.3.0-alpine/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:10.3.0-alpine
+FROM node:10.5.0-alpine
```

Comment:
Are these generated files?  Did you update them manually or automatically?  The file path (generated-dockerfiles/alpine/10.3.0-alpine/Dockerfile) doesn't match the version 10.5.0.

Similar feedback for the other Dockerfiles.

Reply:
Right. I updated manually. Didn't realize there was a script to update these stuff for us. I run the script now and updated the PR. Should be all good now.
Should we add a `CONTRIBUTING.md` (or just a section to the README.md) for describing how to update Node docker images?